extensions/api/auth-basic/src/test/java/org/eclipse/dataspaceconnector/api/auth/DummyApiExtension.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2022 ZF Friedrichshafen AG
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Contributors:
+ *    ZF Friedrichshafen AG - Initial API and Implementation
+ */
+
+package org.eclipse.dataspaceconnector.api.auth;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import org.eclipse.dataspaceconnector.spi.WebService;
+import org.eclipse.dataspaceconnector.spi.system.Inject;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtension;
+import org.eclipse.dataspaceconnector.spi.system.ServiceExtensionContext;
+
+public class DummyApiExtension implements ServiceExtension {
+
+    @Inject
+    WebService webService;
+
+    @Override
+    public void initialize(ServiceExtensionContext context) {
+        webService.registerResource("data", new DummyApiController());
```

Comment:
tests are broken because you need to register also an `AuthenticationRequestFilter` with the injected `AuthenticationService` and also the `EdcApiExceptionMapper`.
This could be made more simple, but let's do in another PR